deploy/import-upstream.sh

```diff
@@ -17,12 +17,27 @@ oc import-image prometheus-alertmanager:v0.15.0 --from=quay.io/openshift/origin-
 # Version 1.4-6 here is used to reference the expected downstream version from
 # the template and to provide some additional context for understanding the
 # mapping of upstream to downstream versions of the qdrouterd container image.
-oc import-image amq-interconnect:1.4-7 --from=quay.io/interconnectedcloud/qdrouterd:1.7.0 --confirm
+oc import-image amq-interconnect:1.4-7 --from=quay.io/interconnectedcloud/qdrouterd:1.8.0 --confirm
```

Comment:
I was having issues with the existing version, but that could have been a red herring. I should downgrade this and determine if every thing still works as it should. The upstream version should really be 1.8.0 though since the 1.7.0 is missing some of the functionality that is packaged in the downstream version (which is 1.7 based) but not delivered in the upstream 1.7.0 version.